### DISCUSSION TITLE
How to view log file?

### DISCUSSION THREAD

**Discussion by klaberte** [JunkFood02/Seal#1057] [Q&A]
Using the latest v1.10.0 with stable yt-dlp 2023.07.06, how can a view the log file, i.e. the combination of all of the "Detailed output" shown when that feature is selected in Settings-General-Detailed output?  In particular, I want to see errors of unsuccessful downloads.

**Comment by Marco-9456** [comment]
If an error occurs, the app will print the error message beneath the URL filed.
